1.9.0

```diff
@@ -771,6 +771,186 @@
           }
         }
       ]
+    },
+    {
+      "sObjectName":"Account",
+      "description": "Add new picklist value to Source",
+      "taskList":[
+        {
+          "jiraTicket":"ac-63412a",
+          "actionType":"PicklistValueStandard",
+          "actionTypeName":"AccountSource",
+          "optionsMap":{
+            "operation" : "add",
+            "picklistValues":"Walk In,Twitter,Facebook,Social Media,Customer Referral"
+          }
+        }
+      ]
+    },
```

Comment:
Looks like you only have to add these values to Lead.Source and it will populate Account Source and Opportunity Source since it is shared across objects, its a special sf field. 

![image](https://github.com/user-attachments/assets/9ec35e64-8ff2-494f-adbc-68fb0fb35c20)